tests/test_cmaker.py

```diff
@@ -130,6 +130,56 @@ def test_make(configure_with_cmake_source_dir, capfd):
             assert message in out
 
 
+@pytest.mark.parametrize("target", ("install", "install-runtime", "bad_value"))
```

Comment:
I added the empty string. And it doesn't generate an error, it will take the default target of the maker.

However not sure if this breaks any future assumptions that `maker` performs an install step.